database/user.go

```diff
@@ -30,13 +31,44 @@ const (
 	TierPremium80
 	// TierMaxReserved is a guard value that helps us validate tier values.
 	TierMaxReserved
+
+	// bpsToBytesPerSecond is a multiplier to get from 1 bps to X bytes per sec.
+	bpsToBytesPerSecond = 1024 * 1024 / 8
```

Comment:
What's bps? If it's bits per second then this doesn't seem right.

Reply:
Bad naming. It's meant to be megabits per second.

I renamed it to `mbpsToBytesPerSecond` and updated the comment.